### PR TITLE
Added options to skip interactive prompts during AWS cleanup

### PR DESCRIPTION
**Pull request by clacroix12**
Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

**Comment by clacroix12** [comment]
@ebenahar this change supports [OCSQE-221](https://projects.engineering.redhat.com/browse/OCSQE-221). The interactive prompts for confirming the cleanup as well as for entering the additional prefixes stopped this from being automated easily. This change allows us to skip those prompts and simply run the cleanup.

We might not always want to skip the prefixes in the future, so I will create an issue to take in the additional prefix map as another CLI arg instead of an interactive prompt. This way we can pass that value through from the jenkins job as well if necessary.

File: ocs_ci/cleanup/aws/cleanup.py [hunk 1]
```diff
@@ -131,28 +131,46 @@ def aws_cleanup():
...
-        "{\'prefix1\': 36, \'prefix2\': 48}\" "
-    )
-    assert confirmation == defaults.CONFIRMATION_ANSWER, "Wrong confirmation answer. Exiting"
+    if not args.force:
+        confirmation = input(
+            'Careful! This action could be highly destructive. '
+            'Are you sure you want to proceed? '
+        )
+        assert confirmation == defaults.CONFIRMATION_ANSWER, (
+            "Wrong confirmation answer. Exiting"
```
Comment (vavuthu) [inline ocs_ci/cleanup/aws/cleanup.py]: is it good to display what is the confirmation answer. By seeing the Question, I prefer "y" or "yes" but defaults.CONFIRMATION_ANSWER is different than what I think. 
WDYT?

File: ocs_ci/cleanup/aws/cleanup.py [hunk 2]
```diff
@@ -131,28 +131,46 @@ def aws_cleanup():
         required=False,
         help="The name of the AWS region to delete the resources from"
     )
+    parser.add_argument(
+        '--force',
+        action='store_true',
+        required=False,
+        help="Skip confirmation prompt"
```
Comment (petr-balogh) [inline ocs_ci/cleanup/aws/cleanup.py]: Maybe worth to mention also about destructive operation.

File: ocs_ci/cleanup/aws/cleanup.py (hunk 1 above)
Comment (petr-balogh) [reply ocs_ci/cleanup/aws/cleanup.py]: Coady is just indenting this in the new condition for force param. Doesn't change the logic here. If there is issue with this it should be addressed in different PR. You can open the issue for this.

File: ocs_ci/cleanup/aws/cleanup.py (hunk 2 above)
Comment (clacroix12) [reply ocs_ci/cleanup/aws/cleanup.py]: Done

File: ocs_ci/cleanup/aws/cleanup.py [hunk 3]
```diff
@@ -131,28 +131,47 @@ def aws_cleanup():
         required=False,
         help="The name of the AWS region to delete the resources from"
     )
+    parser.add_argument(
+        '--force',
+        action='store_true',
+        required=False,
+        help="Skip confirmation prompt. WARNING: this utility is destructive, "
```
Comment (shylesh) [inline ocs_ci/cleanup/aws/cleanup.py]: @clacroix12 ,
For me its not clear about what confirmation prompt will be skipped! Can this be more explicit ?

Comment (clacroix12) [reply ocs_ci/cleanup/aws/cleanup.py]: Done

**Pull request merged**